### DISCUSSION TITLE
log 里的 connection reset by peer 是怎么回事？

### DISCUSSION THREAD

**Discussion by bluebigeye** [apernet/hysteria#1101] [Q&A]
用的 v2.4.4，客户端日志里有 connection reset by peer。我的理解是 *墙* 发现了是对 google 的访问，然后对我发送了一个 reset 命令？

```
WARN	SOCKS5 TCP error	{"addr": "127.0.0.1:58214", "reqAddr": "encrypted-tbn3.gstatic.com:443", "error": "read tcp4 127.0.0.1:5000->127.0.0.1:58214: read: connection reset by peer"}
```

这说明我的配置并非完全隐身？不知道我需要修改什么配置才能更加隐秘呢？我使用的是 [Bing自签证书](https://github.com/flame1ce/hysteria2-install/blob/main/hysteria2-install-main/hy2/hysteria.sh#L140C1-L147C29)这段代码 和 `insecure: true`。

**Comment by haruue** [comment]
TCP Reset 除了被 GFW 用来屏蔽网站或许还有些别的用途。

早在 1981 年第一版 TCP 规范文档（[RFC 793](https://www.ietf.org/rfc/rfc793.txt)）中就已经定义 TCP Reset 了， 那个时候中国都还没通网更没有 GFW。
